devenv/datasources.yaml

```diff
@@ -126,7 +126,7 @@ datasources:
   - name: gdev-elasticsearch-v7-metrics
     type: elasticsearch
     access: proxy
-    database: "[metrics-]YYYY.MM.DD"
+    database: "[metricbeat-]YYYY.MM.DD"
```

Comment:
Hmm not sure this is right, there already is a container `gdev-elasticsearch-v7-metricbeat` using this index.

Reply:
Not sure how filling of the data works here but maybe the issue is just that `metrics-` indexes aren't filled by the fake data generator.

Reply:
> Hmm not sure this is right, there already is a datasource gdev-elasticsearch-v7-metricbeat using this index.

u r right, haven't noticed